trieste/acquisition/function/function.py

```diff
@@ -1034,6 +1035,613 @@ def __call__(self, x: TensorType) -> TensorType:
         return tf.reduce_mean(batch_improvement, axis=-1, keepdims=True)  # [..., 1]
 
 
+class BatchExpectedImprovement(SingleModelAcquisitionBuilder[ProbabilisticModel]):
+    """Accurate approximation of the batch expected improvement, using the
+    method of Chvallier and Ginsbourger :cite:`chevalier2013fast`.
+
+    Internally, this uses a highly accurate approximation of the cumulative
+    density function of the multivariate Gaussian, developed by Alan Genz
+    :cite:`genz2016numerical`.
+    """
+
+    def __init__(
+        self,
+        sample_size: int,
+        *,
+        jitter: float = DEFAULTS.JITTER,
+    ):
+        """Initialise the BatchExpectedImprovement instance.
+
+        :param sample_size: int, number of Sobol samples to use.
+        :param jitter: float, amount of jitter for Cholesky factorisations.
+        """
+
+        tf.debugging.assert_positive(sample_size)
+        tf.debugging.assert_greater_equal(jitter, 0.0)
+
+        self._sample_size = sample_size
+        self._jitter = jitter
+
+    def __repr__(self) -> str:
+        """"""
+
+        return f"BatchExpectedImprovement({self._sample_size!r}, " f"jitter={self._jitter!r})"
+
+    def prepare_acquisition_function(
+        self,
+        model: ProbabilisticModel,
+        dataset: Optional[Dataset] = None,
+    ) -> AcquisitionFunction:
+        """ """
```

Comment:
The other AFs include the params in the docstring (and in update_acquisition_function) so probably best to be consistent.